src/app/navbar/navbar.component.scss

```diff
@@ -14,18 +14,34 @@
  *    limitations under the License.
  */
 
+@import 'materialColorScheme.scss';
+
 #dropdown-main {
   font-weight: bold;
   text-decoration: none;
   color: white;
 }
 
 .icon {
-  padding: 0 14px;
+  padding: 14px 14px;
   font-weight: bold;
   text-decoration: none;
   color: white;
   font-size: 14px;
+  border-radius: 0;
+  height: 64px;
+}
+
+.icon:hover {
+  background-color: #112246;
```

Comment:
what is this color? only used for this component and not elsewhere or in the color palette?

Reply:
It's the color for the navigation bar's hover state, according to the [zeplin](https://app.zeplin.io/project/5fc690a2717ee9144814f190/screen/607da9695f1d1e9c06b6e74e) and it's only used in this component. The actual color isn't specified in the zeplin, but using the svg image given, I was able to identify the color as `#112246`. I couldn't find that color in [kim's color palette](https://app.zeplin.io/project/5fc690a2717ee9144814f190/screen/5fc699d068c46f17551f745b)

Reply:
put a code comment and say whether it's darker or lighter than the ones in kim's color palette